### PR TITLE
Fixed bug (cannot reuse hashtags)

### PR DESCRIPTION
**Pull request by d-ugol**
In tweet.rb, changed 
  hashtag = Hashtag.new(tag: tag)
to
  hashtag = Hashtag.find_or_create_by( tag: tag)
so that if the hashtag already exists (is found), then the app will not insert a new record into the hashtags table.

### CODE REVIEW

File: app/models/tweet.rb [hunk 1]
```diff
@@ -7,7 +7,10 @@ class Tweet < ApplicationRecord
 
   def generate_hashtags
     content.scan(/#\w+/).each do |tag|
-      hashtag = Hashtag.new(tag: tag)
+
+      hashtag = Hashtag.find_or_create_by( tag: tag)
+
+      #hashtag = Hashtag.new(tag: tag)
       hashtag.save
```
Comment (nw0428) [inline app/models/tweet.rb]: This save is unnecessary

File: app/models/tweet.rb [hunk 2]
```diff
@@ -7,7 +7,10 @@ class Tweet < ApplicationRecord
 
   def generate_hashtags
     content.scan(/#\w+/).each do |tag|
-      hashtag = Hashtag.new(tag: tag)
+
+      hashtag = Hashtag.find_or_create_by( tag: tag)
+
+      #hashtag = Hashtag.new(tag: tag)
```
Comment (nw0428) [inline app/models/tweet.rb]: Remove unused code please

**Review by nw0428 — COMMENTED** [review]
Really good. Just a few small changes

**Comment by d-ugol** [comment]
Removed .save, removed commented-out code.

**Review by nw0428 — APPROVED** [review]
[LGTM](http://lgtm.in/i/4kBDR9sZp)

**Pull request merged**